dotcom-rendering/src/components/AudioAtom/AudioAtom.tsx

```diff
@@ -0,0 +1,387 @@
+						'timeupdate',
+						updateCurrentTimeAndPosition,
+				  )
+				: undefined;
+	}, [audioEl, setCurrentTime, shouldUseAcast]);
+
+	// update duration time
+	const [durationTime, setDurationTime] = useState<number>(duration);
+	useEffect(() => {
+		const audioElCurrent = audioEl.current;
+		const updateDurationTime = () =>
+			setDurationTime(
+				audioEl.current ? audioEl.current.duration : duration,
+			);
+
+		audioEl.current?.addEventListener('loadeddata', updateDurationTime);
+		return () =>
+			audioElCurrent?.removeEventListener(
+				'loadeddata',
+				updateDurationTime,
+			);
+	}, [audioEl, setDurationTime, duration]);
+
+	const updateAudioCurrentTime = (e: MouseEvent<HTMLInputElement>) => {
+		if (audioEl.current) {
+			const percentagePositionClick =
+				(e.nativeEvent.offsetX / e.currentTarget.offsetWidth) * 100;
+			// set the currentTime of the audio based on percentagePositionClick
+			audioEl.current.currentTime =
+				(audioEl.current.duration / 100) * percentagePositionClick;
+		}
+	};
+
+	// ***************************
+	// *     Accessibility       *
+	// ***************************
+	const progressBarEl = useRef<HTMLInputElement>(null);
+	useEffect(() => {
+		const rightArrowKeyCode = 39;
+		const leftArrowKeyCode = 37;
+		const keyListener = (e: KeyboardEvent) => {
+			if (
+				e.keyCode === rightArrowKeyCode &&
+				document.activeElement === progressBarEl.current
+			) {
+				if (audioEl.current) audioEl.current.currentTime += 15;
+			}
+
+			if (
+				e.keyCode === leftArrowKeyCode &&
+				document.activeElement === progressBarEl.current
+			) {
+				if (audioEl.current) audioEl.current.currentTime -= 15;
+			}
+		};
+
+		document.addEventListener('keydown', keyListener);
+		return () => document.removeEventListener('keydown', keyListener);
+	}, [audioEl, progressBarEl]);
+
+	// If Acast is enabled, replace the default url with an ad enabled one
+	useEffect(() => {
+		setUrlToUse(buildUrl(trackUrl, shouldUseAcast));
+	}, [shouldUseAcast, trackUrl]);
+
+	const playAudio = () => {
+		setIsPlaying(true);
+		void audioEl.current?.play();
+	};
+
+	const pauseAudio = () => {
+		setIsPlaying(false);
+		audioEl.current?.pause();
+	};
+
+	return (
+		<div css={wrapperStyles} data-atom-id={id} data-atom-type="audio">
+			<div
+				css={css`
+					padding-left: 5px;
+				`}
+			>
+				<span css={kickerStyle(pillar)}>{kicker}</span>
+				<h4 css={titleStyle}>{title}</h4>
+			</div>
+			<div css={audioBodyStyle}>
+				{/* eslint-disable-next-line jsx-a11y/media-has-caption -- TODO */}
+				<audio
+					css={audioElementStyle}
+					src={urlToUse}
+					ref={audioEl}
+					preload="none"
+					data-component="inarticle audio"
+					data-duration={durationTime}
+					data-media-id={id ?? '_no_ids'}
+					data-title={titleStyle}
+				>
+					<p>
+						Sorry your browser does not support audio - but you can
+						download here and listen
+						https://audio.guim.co.uk/2020/05/05-61553-gnl.fw.200505.jf.ch7DW.mp3
```

Comment:
I'm curious why this is hardcoded - I wonder if we should be using `urlToUse` instead?

```suggestion
						${urlTouse}
```